.storybook/preview-head.html

```diff
@@ -0,0 +1,6 @@
+<link
+  href="//fast.fonts.net/cssapi/a9c44fdd-9d28-4e84-9c95-c1d2c4f0220a.css"
+  rel="stylesheet"
+  type="text/css"
+/>
+<link rel="stylesheet" href="http://localhost:5000/assets/application.css" />
```

Comment:
This file gets inserted into the `<head>` of Storybook pages so that we get global styles and fonts, giving us visual parity with the running app.

(Note that for this to work properly, you have to be running a Rails server on :5000 in addition to the Storybook server.)